ghost/i18n/locales/hi/portal.json

```diff
@@ -0,0 +1,159 @@
+    "{{amount}} off forever.": "हमेशा के लिए {{amount}} की छूट।",
+    "{{discount}}% discount": "{{discount}}% छूट",
+    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} को अब {{newsletterName}} न्यूज़लेटर प्राप्त नहीं होगा।",
+    "{{memberEmail}} will no longer receive emails when someone replies to your comments.": "{{memberEmail}} को अब आपके कमेंट्स पर कोई उत्तर देने पर ईमेल नहीं मिलेगा।",
+    "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} को अब यह न्यूज़लेटर प्राप्त नहीं होगा।",
+    "{{trialDays}} days free": "{{trialDays}} दिन मुफ्त",
+    "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "एक लॉगिन लिंक आपके इनबॉक्स में भेजा गया है। यदि यह 3 मिनट में नहीं आता है, तो कृपया अपना स्पैम फ़ोल्डर जांचें।",
+    "Account": "खाता",
+    "Account settings": "खाता सेटिंग्स",
+    "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "मुफ्त परीक्षण समाप्त होने के बाद, आपको आपके द्वारा चुने गए स्तर के लिए नियमित कीमत पर शुल्क लिया जाएगा। आप तब तक किसी भी समय रद्द कर सकते हैं।",
+    "Already a member?": "पहले से ही सदस्य हैं?",
+    "An unexpected error occured. Please try again or <a>contact support</a> if the error persists.": "एक अप्रत्याशित त्रुटि हुई। कृपया फिर से प्रयास करें या यदि त्रुटि बनी रहती है तो <a>समर्थन से संपर्क करें</a>।",
+    "Back": "वापस",
+    "Back to Log in": "लॉगिन पर वापस जाएं",
+    "Billing info": "बिलिंग जानकारी",
+    "Black Friday": "ब्लैक फ्राइडे",
+    "Cancel anytime.": "कभी भी रद्द करें।",
+    "Cancel subscription": "सदस्यता रद्द करें",
+    "Cancellation reason": "रद्द करने का कारण",
+    "Change": "परिवर्तन",
+    "Check spam & promotions folders": "स्पैम और प्रचार फ़ोल्डरों की जांच करें",
+    "Check with your mail provider": "अपने मेल प्रदाता से जांचें",
+    "Choose": "चुनें",
+    "Choose a different plan": "एक अलग योजना चुनें",
+    "Choose your newsletters": "अपने न्यूज़लेटर चुनें",
+    "Click here to retry": "फिर से प्रयास करने के लिए यहां क्लिक करें",
+    "Close": "बंद करें",
+    "Comments": "टिप्पणियाँ",
+    "Complimentary": "नि:शुल्क",
+    "Confirm": "पुष्टि करें",
+    "Confirm cancellation": "रद्दीकरण की पुष्टि करें",
+    "Confirm subscription": "सदस्यता की पुष्टि करें",
+    "Contact support": "समर्थन से संपर्क करें",
+    "Continue": "जारी रखें",
+    "Continue subscription": "सदस्यता जारी रखें",
+    "Could not sign in. Login link expired.": "साइन इन नहीं कर सके। लॉगिन लिंक समाप्त हो गया।",
+    "Could not update email! Invalid link.": "ईमेल अपडेट नहीं कर सके! अमान्य लिंक।",
+    "Create a new contact": "एक नया संपर्क बनाएं",
+    "Current plan": "वर्तमान योजना",
+    "Delete account": "खाता हटाएं",
+    "Didn't mean to do this? Manage your preferences <button>here</button>.": "यह करने का मतलब नहीं था? अपनी प्राथमिकताएँ <button>यहाँ</button> प्रबंधित करें।",
```

Comment:
यह करने का मतलब नहीं था? doesn't sound right
Maybe this: क्या ऐसा करने का इरादा नहीं था?

Reply:
Updated